web/src/components/users/RootAuthMethodsPage.tsx

```diff
@@ -0,0 +1,157 @@
+
+    const data: Partial<RootUserChanges> = {};
+
+    if (!isEmpty(password)) {
+      data.password = password;
+      data.encryptedPassword = false;
+    }
+
+    if (!isEmpty(sshKey)) {
+      data.sshkey = sshKey;
+    }
+
+    if (isEmpty(data)) return;
+
+    await setRootUser.mutateAsync(data);
+
+    navigate(PATHS.root);
+  };
+
+  // TRANSLATORS: %s will be replaced by a link with the text "upload".
+  const [sshKeyStartHelperText, sshKeyEndHelperText] = _(
+    "Write, paste, drop, or %s a SSH public key file in the above textarea.",
+  ).split("%s");
+
+  return (
+    <Page>
+      <Page.Content>
+        <Center>
+          <Page.Section
+            headerLevel="h2"
+            title={_("Setup root user authentication")}
+            description={
+              <p className={textStyles.fontSizeXl}>
+                {_(
+                  "You must define at least one authentication method for the root user. You can still edit them anytime before the installation.",
+                )}
+              </p>
+            }
+            pfCardProps={{ isCompact: false }}
+            pfCardBodyProps={{ isFilled: true }}
+          >
+            <Form id="rootAuthMethods" onSubmit={accept}>
+              <Flex direction={{ default: "column" }} rowGap={{ default: "rowGapXl" }}>
+                <FormGroup fieldId="rootPassword" label={_("Password")}>
+                  <PasswordInput
+                    inputRef={passwordRef}
+                    id="rootPassword"
+                    value={password}
+                    className={sizingStyles.wAuto}
+                    onChange={(_, value) => setPassword(value)}
+                  />
+                </FormGroup>
+                <FormGroup fieldId="sshKey" label={_("SSH public key")}>
+                  <FileUpload
+                    id="sshKey"
+                    value={sshKey}
+                    type="text"
+                    aria-label={_(
+                      "Write, paste, or drop an SSH public key here. You can also upload it by using the link below.",
+                    )}
+                    // TRANSLATORS: push button label
+                    browseButtonText={_("Upload")}
+                    // TRANSLATORS: push button label, clears the related input field
+                    clearButtonText={_("Clear")}
+                    isLoading={isUploading}
+                    onDataChange={(_, value) => setSSHKey(value)}
+                    onTextChange={(_, value) => setSSHKey(value)}
+                    onReadStarted={startUploading}
+                    onReadFinished={stopUploading}
+                    onClearClick={clearKey}
+                  />
+                  <FormHelperText>
+                    <HelperText>
+                      <HelperTextItem variant="indeterminate">
+                        {sshKeyStartHelperText}
+                        <Button variant="link" isInline onClick={uploadFile}>
+                          {_("upload")}
```

Comment:
Here a TRANSLATORS comment is needed, or they will not know whether to use a noun or verb, in which form
("*sube* el archivo" vs "*carga* de archivo completada")

I'm not sure of the correct syntax for the comment, please adapt and insert:

// TRANSLATORS: this "upload" is a commanding verb, in the %s place of
// "Write, paste, drop, or %s a SSH public key file in the above textarea."

Reply:
Thank you!